src/index.js

```diff
@@ -159,9 +204,17 @@ class HawkWebpackPlugin {
           filename: map.name
         });
         body.append('release', releaseId);
-        body.append('commits', []);
 
-        // this.log(`Sending map [${map.name}] ...`)
+        if (this.commits) {
+          try {
+            const commits = this.findCommits() || [];
```

Comment:
but what if there are no commits? or no git. `commits` does not checked for emptiness